src/itest/kotlin/nl/lifely/zac/itest/client/ItestHttpClient.kt

```diff
@@ -161,6 +168,8 @@ class ItestHttpClient {
             .add("Content-Type", "application/json")
             .build()
 
-    private fun cloneHeadersWithAuthorization(headers: Headers): Headers =
-        headers.newBuilder().add("Authorization", "Bearer ${KeycloakClient.requestAccessToken()}").build()
+    private fun cloneHeadersWithAuthorization(headers: Headers, openZaakAuth: Boolean = false): Headers {
```

Comment:
Or can you determine from the URL what auth header should be set? That might even be more "flexible", in case other URLs have to be invoked in the future?

Reply:
We have to check for the port then and hardcode 8001. Not sure this is much better

Reply:
Checking the port now